app/src/main/res/layout/fragment_forms.xml

```diff
@@ -27,21 +27,20 @@
 
         <ImageView
             android:id="@+id/syncIcon"
-            android:layout_width="@dimen/small_icon_size"
-            android:layout_height="@dimen/small_icon_size"
-            android:contentDescription="@null"
+            android:layout_width="@dimen/medium_icon_size"
+            android:layout_height="@dimen/medium_icon_size"
             android:layout_marginEnd="@dimen/small_margin"
+            android:contentDescription="@null"
             app:layout_constraintBottom_toBottomOf="@id/syncInfo"
             app:layout_constraintEnd_toStartOf="@id/syncInfo"
             app:layout_constraintHorizontal_chainStyle="packed"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="@id/syncInfo"
-            app:layout_constraintVertical_bias="0"
```

Comment:
In the pictures that you attached  this icon sits at the top in relation to the text so I used this attribute to push it up. If it 's acceptable without the attribute then leave it out.

Reply:
oh, yes. you are right, we can leave the image at the top :)